docs/src/manual/solutions.md

```diff
@@ -625,9 +625,19 @@ To use the primal solution from a solve, omit the `point` argument:
 ```jldoctest feasibility
 julia> optimize!(model)
 
-julia> primal_feasibility_report(model)
+julia> primal_feasibility_report(model; atol = 0.0)
 Dict{Any, Float64}()
 ```
+Calling [`primal_feasibility_report`](@ref) without the `point` argument is most
+useful when the solver reports that the solutionn is feasible, but you want to
```

Comment:
I'm not sure this is true. It could also be used for first-order solvers that return approximate solutions or stopped early with unknown solution quality.